node/config/types.go

```diff
@@ -542,10 +542,6 @@ type FevmConfig struct {
 	// This will also enable the RealTimeFilterAPI and HistoricFilterAPI by default, but they can be disabled by config options above.
```

Comment:
look at the line above this one - the text is now wrong, in fact, `EnableEthRPC` without ChainIndexer is going to be a big problem and will likely be unusable because of the lack of the tx<>msgcid mapping

Reply:
Fixed in https://github.com/filecoin-project/lotus/pull/12421/commits/f628cb2ac04ec539f3d31ee6ffb6034c79aed156.

Reply:
Line above this one still needs fixing, the text says:

```
	// EnableEthRPC enables eth_ rpc, and enables storing a mapping of eth transaction hashes to filecoin message Cids.
	// This will also enable the RealTimeFilterAPI and HistoricFilterAPI by default, but they can be disabled by config options above.
```

Could become:

```
	// EnableEthRPC enables eth_ RPC methods and will also enable the RealTimeFilterAPI.
	// Setting this to true will also require that ChainIndexer is enabled, otherwise it will cause an error at startup.
```

Reply:
This is done.